src/main/webapp/WEB-INF/pages/checkoutPage.jsp

```diff
@@ -0,0 +1,116 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
```

Comment:
orderOverviewPage.jsp and checkoutPage.jsp files have duplicate code fragments. For the future: think of how to avoid this.